resources/public/api/conf/1.0/application.yaml

```diff
@@ -18,32 +41,28 @@ components:
       schema:
         type: string
         enum: [
-          "application/vnd.hmrc.1.0+json",
-          "application/vnd.hmrc.1.0+xml"
+            "application/vnd.hmrc.1.0+json",
```

Comment:
Usually in our application.yaml files under components we have the `securitySchemes`. The headers are normally specified under each individual endpoint yaml file so probably dont need this section